PowerEditor/installer/nsisInclude/gobalDef.nsh

```diff
@@ -34,6 +34,12 @@
 !define VERSION_MAJOR 7
 !define VERSION_MINOR 0
 
+!define CompanyName		"Don HO don.h@free.fr"
+!define Description		"Notepad++ : a free (GNU) source code editor"
+!define Version 		"7.0.0.0"
```

Comment:
This is yet another place to modify when a new version comes out. Why not use the macros above?